<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MYAPP</class>
 <widget class="QWidget" name="MYAPP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SIC IoT - CapstoneProjectApp</string>
  </property>
  <widget class="QLabel" name="SAMSUNGLOGO">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>SAMSUNG_LOGO_w128.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="SIC_LOGO">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>INNOVATION CAMPUS</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="project_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>HỆ THỐNG GIÁM SÁT VÀ CẢNH BÁO CHÁY CHO CĂN HỘ CHUNG CƯ.</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Widget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>341</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Widget</string>
   </property>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="temp_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Temperature:</string>
    </property>
   </widget>
   <widget class="QLabel" name="humid_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Humidity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="temp_value">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>n/a</string>
    </property>
   </widget>
   <widget class="QLabel" name="humid_value">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>n/a</string>
    </property>
   </widget>
   <widget class="QLabel" name="CO2_value">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>n/a</string>
    </property>
   </widget>
   <widget class="QLabel" name="CO2_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CO2:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RefreshButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>361</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera</string>
   </property>
   <widget class="QLabel" name="Camera_Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>301</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>No video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="People_Detection">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>People Detection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Camera_Control">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Start/Stop Camera</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Announcement">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>361</width>
     <height>91</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>System Status</string>
   </property>
   <widget class="QLabel" name="Notification1_Value">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;notification 1&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="ServerConnection_Title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Server Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ServerConnection_Value">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnected</string>
    </property>
   </widget>
   <widget class="QLabel" name="Notification2_Value">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;notification 2&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="sysconf">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>380</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>System configuration</string>
   </property>
   <widget class="QPushButton" name="RebootButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Reboot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FullScreenButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>FullScreen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ServerSyncButton">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ServerSync</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetResetFireAlert_Button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Set/Reset FireAlert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearNotiButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Notification</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="FireWarningBar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>red.jpg</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
